tests/end2end/features/fileselect.feature

```diff
@@ -0,0 +1,35 @@
+# vim: ft=cucumber fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
For every `.feature` file, a new instance of qutebrowser is spanwed (which then runs all the tests in that file). This is to have some isolation between tests, but still have acceptable performance. Because we aren't going to have many tests here, I'd prefer having them in `editor.feature` instead of having them in their own file. It's not a perfect match of course, but I'd say they fit in best there.

Reply:
Merged with `editor.feature`